client/lib/directly/index.js

```diff
@@ -14,45 +20,64 @@ const DEFAULT_RTM_WIDGET_OPTIONS = {
 };
 const DIRECTLY_RTM_SCRIPT_URL = 'http://widgets.wp.com/directly/embed.js';
 const DIRECTLY_ASSETS_BASE_URL = 'https://www.directly.com';
-let initializationPromise;
+let _initialize;
+
+/**
+ * Set up global variables and configuration for the Directly library.
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S
+ */
+function configureDirectlyGlobals() {
+	// Set up the global DirectlyRTM function, required for the Directly library.
+	// This snippet is pasted from Directly's setup code.
+	window.DirectlyRTM = window.DirectlyRTM || function() {
+		( window.DirectlyRTM.cq = window.DirectlyRTM.cq || [] ).push( arguments );
+	};
+	// Since we can only configure once per pageload, this library only provides a
+	// single global configuration.
+	window.DirectlyRTM( 'config', DEFAULT_RTM_WIDGET_OPTIONS );
+}
 
+/**
+ * Inserts a dummy DOM element that Directly uses to calculate the base URL for its assets.
+ *
+ * Under standard setup, Directly gathers its base URL by inspecting the src attribute of
+ * the <script> used to load the library. Since we've got a custom setup that doesn't include
+ * a <script> tag, we need to fake this by placing a DOM element with id="directlyRTMScript"
+ * and a src with the appropriate base URL.
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S
+ */
 function insertDirectlyDOM() {
-	// Directly gathers the base URL for its assets by inspecting the src attribute
-	// of a DOM element with id="directlyRTMScript". Since we're not using the standard
-	// integration, we need to fake it by adding a dummy DOM object with those attributes.
 	const d = document.createElement( 'div' );
 	d.id = 'directlyRTMScript';
 	d.src = DIRECTLY_ASSETS_BASE_URL;
 	document.body.appendChild( d );
 }
 
+/**
+ * Initializes Directly if it hasn't already been initialized, and then executes the
+ * command by passing the arguments to window.DirectlyRTM
+ *
+ * @see initialize
+ *
+ * @returns {Promise} Promise that resolves after initialization and command execution
+ */
 function executeDirectlyCommand( ...args ) {
```

Comment:
see comment about naming above. if we are trying to namespace-by-function-prefix we'll end up with stuff like this…

```js
directly.executeDirectlyCommand
```

…instead of this…

```js
directly.execute()

// or

directly.exeucuteCommand()
```